Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 
 name = "underscore"
-version = "0.0.2"
+version = "0.0.21"
```

Comment:
Ah, you are right. I had overlooked.
But I think there is no problem to work.
cf: I think version number should follow [Semantic Versioning](http://semver.org/).